### PR TITLE
Added Slack Community Announcement Bar

### PR DESCRIPTION
**Pull request by Tacktician**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -27,6 +27,13 @@ const docusaurusConfig = {
         },
     ],
     themeConfig: {
+        announcementBar: {
+            id: 'site_announcement', // Any value that will identify this message.
+            content: `<a target="_blank" rel="noopener noreferrer" href="https://saucelabs.com/resources/community/slack">\n' + Have a question? Join our <img alt="slack.png" src="/img/slack.png" width="17" height="17" /> Slack Community Workspace!'</a>`,
```
Comment (aliciakrebs) [inline docusaurus.config.js:32]: looks like the single quotes are not needed. 
```suggestion
            content: `<a target="_blank" rel="noopener noreferrer" href="https://saucelabs.com/resources/community/slack">\n + Have a question? Join our <img alt="slack.png" src="/img/slack.png" width="17" height="17" /> Slack Community Workspace!</a>`,
```

File: docusaurus.config.js [hunk 2]
```diff
@@ -27,6 +27,13 @@ const docusaurusConfig = {
         },
     ],
     themeConfig: {
+        announcementBar: {
+            id: 'site_announcement', // Any value that will identify this message.
+            content: `<a target="_blank" rel="noopener noreferrer" href="https://saucelabs.com/resources/community/slack">\n + Have a question? Join our <img alt="slack.png" src="/img/slack.png" width="17" height="17" /> Slack Community Workspace!</a>`,
```
Comment (lysannep) [inline docusaurus.config.js:32]: Is the "+" needed?

Comment (lysannep) [inline docusaurus.config.js:32]: Suggestion to remove the "+" and move the Slack icon to the beginning of the paragraph. We could also change the text to say "Join us on SLack". 

```suggestion
            content: `<a target="_blank" rel="noopener noreferrer" href="https://saucelabs.com/resources/community/slack"><img alt="slack.png" src="/img/slack.png" width="17" height="17" /> Have a question? Join us on Slack!</a>`,
```
Screenshot 2023-05-19 at 1 30 59 PM

**Review by lysannep — COMMENTED** [review]
I added a suggestion. Thanks!

**Review by aliciakrebs — APPROVED** [review]

**Pull request closed without merging**